deploy/charts/cert-manager/README.template.md

```diff
@@ -148,6 +151,9 @@ The following table lists the configurable parameters of the cert-manager chart
 | `cainjector.podAnnotations` | Annotations to add to the cainjector pods | `{}` |
 | `cainjector.deploymentAnnotations` | Annotations to add to the cainjector deployment | `{}` |
 | `cainjector.extraArgs` | Optional flags for cert-manager cainjector component | `[]` |
+| `cainjector.serviceAccount.create` | If `true`, create a new service account for the webhook cainjector | `true` |
```

Comment:
`... account for the webhook cainjector` -> `account for the webhook component`

Reply:
`account for the webhook component`? the cainjector isn't part of the webhook so shouldn't it actually be `account for the cainjector component`?

Reply:
Yep 🤦